tests/test_efftox_integration.py

```
@@ -0,0 +1,93 @@
+    priors = [norm(0, 1) for _ in range(6)]
+    scaled_doses = [-1.0, 0.0, 1.0]
+
+    # 20 patients at dose 3, all had Tox and no Eff
+    cases = [(3, 1, 0)] * 20
+    tox_cutoff = 0.3
+    eff_cutoff = 0.5
+
+    # Run with small max_iter and large mass_threshold to force a warning or expansion
+    # We use a smaller n for speed in tests
+    with caplog.at_level(logging.WARNING):
+        probs, pds = efftox_get_posterior_probs(
```

Ruff (0.15.18)

[warning] Unpacked variable `probs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)